pkg/edgeadm/constant/manifests/application-grid-wrapper.go

```diff
@@ -127,14 +127,16 @@ spec:
       labels:
         k8s-app: application-grid-wrapper-node
     spec:
+      tolerations:
+      - operator: Exists
```

Comment:
Why do we need this toleration?

Reply:
The application-grid-wrapper may be more important than other superedge app, it can tolerate any taint in node, otherwise it can be drained and pod service forward will failed